testscript/cmd.go

```diff
@@ -48,6 +48,15 @@ var scriptCmds = map[string]func(*TestScript, bool, []string){
 	"wait":     (*TestScript).cmdWait,
 }
 
+var builtinCmds map[string]bool
```

Comment:
Rather than doing this, how about just putting the check into the actual command function itself? That way it'll work even if the command overrides a builtin command too.

In `exe.go`:
```
			scriptCmds[name] = func(ts *TestScript, neg bool, args []string) {
				if ts.params.RequireExplicitExec {
					ts.Fatalf("use 'exec %s' rather than '%s' (because RequireExplicitExec is enabled)", args[0], args[0])
				}
				ts.cmdExec(neg, append([]string{name}, args...))
			}
```

(also above, a mild suggestion for a phrasing change to the error message because i'm not overly keen on "as per")

Reply:
Much cleaner, thank you!

I took this route because, in my head, builtins could not be shadowed or overriden. That's a question for a different thread, though.